examples/systemcalls/systemcalls.c

```diff
@@ -16,7 +20,10 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
-
+    int ret = system (cmd);
+    if( ret  == -1)
```

Comment:
@jasan2303  think about what would happen if the cmd returned with an error code other than -1 here.